lib/Backend/Lower.cpp

```diff
@@ -12936,7 +12935,7 @@ Lowerer::LowerInlineeEnd(IR::Instr *instr)
     {
         // REVIEW (michhol): OOP JIT. why are we creating an addropnd with 0?
```

Comment:
sounds good to me

---
In reply to: [80370964](https://github.com/Microsoft/ChakraCore/pull/1630#discussion_r80370964) [](ancestors = 80370964)